runtime/src/bank.rs

```diff
@@ -423,31 +423,24 @@ impl Bank {
         self.status_cache.write().unwrap().clear_signatures();
     }
 
+    pub fn is_tx_committable(result: &Result<()>) -> bool {
```

Comment:
Might need another word here. If InstructionError, the accounts should not be committed. Also, please don't use "tx" in function names (and it's the accounts that are committed, not the transaction).

Reply:
And why `pub`?